django_geosource/models.py

```diff
@@ -129,26 +131,39 @@ def refresh_data(self):
             layer = self.get_layer()
             begin_date = datetime.now()
             row_count = 0
```

Comment:
Create an empty report here and add item to it lateron.

Reply:
done